site/content/en/docs/technical-outcomes/scalability/scale-using-vpa/scale-using-vpa.demo

```diff
@@ -0,0 +1,38 @@
+# Scale pod CPU request up & down using VPA
+
+# Get metrics usage
+wget https://github.com/kubernetes-sigs/kwok/releases/download/v0.6.0/metrics-usage.yaml
+
+# Set up Cluster
+kwokctl create cluster --enable-metrics-server --config ${WORK_DIR}/kwokctl.yaml --config ./metrics-usage.yaml --runtime binary
+
+# Create a node
+kwokctl scale node --replicas 1 --param '.allocatable.cpu="4000m"'
+
+# Set up VPA
+git clone https://github.com/kubernetes/autoscaler.git
+kubectl apply -f ./autoscaler/vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml
+kubectl apply -f ./autoscaler/vertical-pod-autoscaler/deploy/vpa-rbac.yaml
+
+# Deploy the three components of the Vertical Pod Autoscaler
+{ cd autoscaler/vertical-pod-autoscaler/ && NAMESPACE=kube-system go run ./pkg/admission-controller --kubeconfig ~/.kube/config --client-ca-file ~/.kwok/clusters/kwok/pki/ca.crt --tls-cert-file  ~/.kwok/clusters/kwok/pki/admin.crt --tls-private-key  ~/.kwok/clusters/kwok/pki/admin.key --webhook-address https://127.0.0.1 --webhook-port 8080 --register-by-url --port 8080 ;} &
+{ cd autoscaler/vertical-pod-autoscaler/ && NAMESPACE=kube-system go run ./pkg/recommender --kubeconfig ~/.kube/config ;} &
+{ cd autoscaler/vertical-pod-autoscaler/ && NAMESPACE=kube-system go run ./pkg/updater --kubeconfig ~/.kube/config ;} &
```

Comment:
```suggestion
{ cd ${ROOT_DIR}/autoscaler/vertical-pod-autoscaler/ && NAMESPACE=kube-system go run ./pkg/admission-controller --kubeconfig ~/.kube/config --client-ca-file ~/.kwok/clusters/kwok/pki/ca.crt --tls-cert-file  ~/.kwok/clusters/kwok/pki/admin.crt --tls-private-key  ~/.kwok/clusters/kwok/pki/admin.key --webhook-address https://127.0.0.1 --webhook-port 8080 --register-by-url --port 8080 ;} &
{ cd ${ROOT_DIR}/autoscaler/vertical-pod-autoscaler/ && NAMESPACE=kube-system go run ./pkg/recommender --kubeconfig ~/.kube/config ;} &
{ cd ${ROOT_DIR}/autoscaler/vertical-pod-autoscaler/ && NAMESPACE=kube-system go run ./pkg/updater --kubeconfig ~/.kube/config ;} &
```

Reply:
Ok, so this means that I would have already cloned the autoscaler repo into this VPA directory, right?